src/main/java/rx/internal/operators/OperatorToObservableSortedList.java

```diff
@@ -100,13 +103,8 @@ public void onNext(T value) {
         child.setProducer(producer);
         return result;
     }
-    // raw because we want to support Object for this default
-    @SuppressWarnings("rawtypes")
-    private static Comparator DEFAULT_SORT_FUNCTION = new DefaultComparableFunction();
 
     private static class DefaultComparableFunction implements Comparator<Object> {
-        DefaultComparableFunction() {
```

Comment:
If you already have that rule written maybe we can apply it? 

* http://pmd.sourceforge.net/pmd-4.3/howtowritearule.html
* https://docs.gradle.org/current/dsl/org.gradle.api.plugins.quality.PmdExtension.html#org.gradle.api.plugins.quality.PmdExtension:ruleSetConfig